dcos_installer/exhibitor_migrate.py

```diff
@@ -127,7 +118,7 @@ def perform():
             print("Migration already complete")
             sys.exit(0)
         else:
-            assert_can_migrate(config)
+            validate_config(config)
```

Comment:
So the expected values are different between these two functions (e.g. `validate_config()` uses `LOG_INDEX_DIR_GOAL` whereas `assert_can_migrate()` uses `LOG_INDEX_DIR_START`). Seems like different functionality... but if `validate_config()` is what we actually want then we should also clean up `LOG_INDEX_DIR_START` and friends since they don't seem to be used anymore.